docs/logfire.md

```diff
@@ -90,65 +96,160 @@ We can also query data with SQL in Logfire to monitor the performance of an appl
+!!! tip ""F**k you, show me the prompt.""
+    As per Hamel Husain's influential 2024 blog post ["Fuck You, Show Me The Prompt."](https://hamel.dev/blog/posts/prompt/)
+    (bare with the capitalization, the point is valid), it's often useful to be able to view the raw HTTP requests and responses made to model providers.
 
-Instrumentation is as easy as adding the following three lines to your application:
+To observer raw HTTP requests made to model providers, you can use `logfire`'s [HTTPX instrumentation](https://logfire.pydantic.dev/docs/integrations/http-clients/httpx/) since all provider SDKs use the [HTTPX](https://www.python-httpx.org/) library internally.
 
-```py {title="instrument_httpx.py" test="skip" lint="skip"}
-import logfire
-logfire.configure()
-logfire.instrument_httpx(capture_all=True)  # (1)!
+
+=== "With HTTP instrumentation"
+
+    ```py {title="with_logfire_instrument_httpx.py" hl_lines="6"}
+    import logfire
+
+    from pydantic_ai import Agent
+
+    logfire.configure()
+    logfire.instrument_pydantic_ai()
+    logfire.instrument_httpx(capture_all=True)  # (1)!
+    agent = Agent('openai:gpt-4o')
+    result = agent.run_sync('What is the capital of France?')
+    print(result.output)
+    #> Paris
+    ```
+
+    1. See the [`logfire.instrument_httpx` docs][logfire.Logfire.instrument_httpx] more details, `capture_all=True` means both headers and body are captured for both the request and response.
+
+    ![Logfire with HTTPX instrumentation](img/logfire-with-httpx.png)
+
+=== "Without HTTP instrumentation"
+
+    ```py {title="without_logfire_instrument_httpx.py"}
+    import logfire
+
+    from pydantic_ai import Agent
+
+    logfire.configure()
+    logfire.instrument_pydantic_ai()
+
+    agent = Agent('openai:gpt-4o')
+    result = agent.run_sync('What is the capital of France?')
+    print(result.output)
+    #> Paris
+    ```
+
+    ![Logfire without HTTPX instrumentation](img/logfire-without-httpx.png)
+
+## Using OpenTelemetry
+
+PydanticAI's instrumentation uses [OpenTelemetry](https://opentelemetry.io/), which Logfire is based on.
+
+This means you can debug and monitor PydanticAI with any OpenTelemetry backend.
+
+PydanticAI follows the [OpenTelemetry Semantic Conventions for Generative AI systems](https://opentelemetry.io/docs/specs/semconv/gen-ai/), so while we think you'll have the best experience using the Logfire platform :wink:, you should get the best experience possible with other platforms.
+
+### Logfire with an alternative OTel backend
+
+You can use the Logfire SDK completely freely and send the data to any OpenTelemetry backend.
+
+Here's an example of configuring the logfire library to send data to the excellent [otel-tui](https://github.com/ymtdzzz/otel-tui) — an open source terminal based OTel backend and viewer (no association with Pydantic).
```

Comment:
It's not clear to me that otel-tui is a better recommendation than jaeger

<img width="1641" alt="Screenshot 2025-05-02 at 18 01 57" src="https://github.com/user-attachments/assets/5546d458-ef35-44fa-b017-88e837452cf0" />

Reply:
I prefer it 🤷, I don't think it matters much.